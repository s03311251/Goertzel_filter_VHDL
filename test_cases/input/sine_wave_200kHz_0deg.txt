2000
3E6E
32CE
0D31
0191
1FFF
3E6E
32CE
0D31
0191
1FFF
3E6E
32CE
0D31
0191
1FFF
3E6E
32CE
0D31
0191
1FFF
3E6E
32CE
0D31
0191
1FFF
3E6E
32CE
0D31
0191
1FFF
3E6E
32CE
0D31
0191
1FFF
3E6E
32CE
0D31
0191
1FFF
3E6E
32CE
0D31
0191
1FFF
3E6E
32CE
0D31
0191
1FFF
3E6E
32CE
0D31
0191
1FFF
3E6E
32CE
0D31
0191
1FFF
3E6E
32CE
0D31
0191
1FFF
3E6E
32CE
0D31
0191
1FFF
3E6E
32CE
0D31
0191
1FFF
3E6E
32CE
0D31
0191
1FFF
3E6E
32CE
0D31
0191
2000
3E6E
32CE
0D31
0191
1FFF
3E6E
32CE
0D31
0191
1FFF
3E6E
32CE
0D31
0191
