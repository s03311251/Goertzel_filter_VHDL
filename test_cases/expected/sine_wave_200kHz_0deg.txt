00000100
000000C0
